src/types/handlers.ts

```diff
@@ -14,3 +14,10 @@ export type Handler = {
   action: ActionHandler[];
   post: PostActionHandler[];
 };
+
+// UserCommandsSchema
+export type UserCommandsSchema = {
```

Comment:
The name looks wrong to me. If this is `type`, it needs to be like `UsersCommands`. Also I think `HelpCommands` sounds more reasonable.